lib/mix/tasks/ash_phoenix.gen.html.ex

```diff
@@ -0,0 +1,113 @@
+defmodule Mix.Tasks.AshPhoenix.Gen.Html do
+  use Mix.Task
+
+  @shortdoc "Generates a controller and HTML views for an existing Ash resource."
+
+  @moduledoc """
+  This task renders .ex and .heex templates and copies them to specified directories.
+
+  ## Arguments
+
+  --api         The API (e.g. "Shop").
```

Comment:
Can we align these arguments with `ash_phoenix.gen.live`? I'm fine to change either the arguments here or the arguments there. I can see a very good case for using only flags like this and not using any positional arguments.

Reply:
I had the same thought. I'd like to postpone a decision about that a bit until the generator eco system has become a bit better. While working on https://elixir-phoenix-ash.com/ash/index.html I constantly realise that basic setup steps take too long. We need more generators to get rid of this barrier. Consolidating them is the following step.

Reply:
I think we increase the surface area for mistakes and the support burden by making them different. If we want to avoid changing the other one then we'd need to make api and resource positional like it is in the other task. I wouldn't want to put this in tutorials and then soon after change its argument signature.

Reply:
I understand. I'll have a look at it.

BTW: I just added https://github.com/ash-project/ash/pull/735 which is much better. I want both to work hand in hand.

Reply:
I changed the UI to `mix ash_phoenix.gen.html Shop Product products`

Now one can do:

```bash
mix ash.gen.resource Shop Product products name:string price:integer
mix ash_phoenix.gen.html Shop Product products
```